tests/integ/test_udf.py

```diff
@@ -1519,20 +1558,13 @@ def test_add_packages_negative(session, caplog):
         session.remove_package("python-dateutil")
     assert "is not in the package list" in str(ex_info)
 
-    with pytest.raises(ValueError) as ex_info:
-        session.add_packages("xgboost==0.1.0")
-    assert "xgboost==0.1.0 because it is not available in Snowflake." in str(ex_info)
-
-    session.clear_packages()
-
 
 @pytest.mark.skipif(
     (not is_pandas_and_numpy_available) or IS_IN_STORED_PROC,
     reason="numpy and pandas are required",
 )
 def test_add_requirements(session, resources_path):
     test_files = TestFiles(resources_path)
-    session.clear_packages()
 
```

Comment:
we no longer need this because `clean_up` does it automatically, right?

Reply:
Yeah